<ui version="4.0" >
 <class>Monitor_UI</class>
 <widget class="QWidget" name="Monitor_UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QFrame" name="video_frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QFrame" name="ruler_frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="KPushButton" name="button_rew" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="button_rew1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="button_play" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="button_fwd1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="button_fwd" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_20" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="KPushButton" name="kpushbutton_9" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_10" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_11" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_12" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_13" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_14" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1" >
    <widget class="KRestrictedLine" name="monitor_time" >
     <property name="inputMask" >
      <string>99:99:99:99; </string>
     </property>
     <property name="validChars" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KRestrictedLine</class>
   <extends>KLineEdit</extends>
   <header>krestrictedline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
